### PR TITLE
Update packages with vulnerabilities

### PR DESCRIPTION
**Pull request by michplunkett**
## Description of Changes
Updating various packages with notable vulnerabilities.

## Tests and Linting
- [x] This branch is up-to-date with the `develop` branch.
- [x] `pytest` passes on my local development environment.
- [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,10 +10,10 @@ Bootstrap-Flask==2.4.0
 boto3==1.28.1
 botocore==1.31.1
 cachetools==5.3.1
-certifi~=2023.5.7
+certifi~=2023.7.22
```
Comment (michplunkett) [inline requirements.txt:13]: Screenshot 2024-10-22 at 3 54 11 PM

File: requirements.txt [hunk 2]
```diff
@@ -10,10 +10,10 @@ Bootstrap-Flask==2.4.0
...
 cffi~=1.15.1
 click==8.1.4
-cryptography~=41.0.1
+cryptography~=43.0.1
```
Comment (michplunkett) [inline requirements.txt:16]: 379015071-eb8cb709-8039-4d56-bea8-5ff172f02b53

File: requirements.txt [hunk 3]
```diff
@@ -32,7 +32,7 @@ Flask-WTF==1.1.2
 google-api-python-client==2.92.0
 gunicorn==22.0.0
 httplib2==0.22.0
-idna~=3.4
+idna~=3.7
```
Comment (michplunkett) [inline requirements.txt:35]: Screenshot 2024-10-22 at 3 55 46 PM

File: requirements.txt [hunk 4]
```diff
@@ -60,7 +60,7 @@ selenium~=4.10.0
 six~=1.16.0
 sphinx==7.1.2
 SQLAlchemy==1.4.52
-tornado~=6.3.2
+tornado~=6.4
```
Comment (michplunkett) [inline requirements.txt:63]: Screenshot 2024-10-22 at 4 00 28 PM

File: requirements.txt [hunk 5]
```diff
@@ -60,13 +60,13 @@ selenium~=4.10.0
...
 trio==0.22.1
 urllib3~=1.26.16 # This version is required for other packages to run
 us==3.1.1
 visitor~=0.1.3
 webencodings~=0.5.1
-Werkzeug~=2.3.6
+Werkzeug~=3.0.1
```
Comment (michplunkett) [inline requirements.txt:69]: Screenshot 2024-10-22 at 4 01 30 PM

File: requirements.txt [hunk 6]
```diff
@@ -43,7 +43,7 @@ MarkupSafe==2.1.3
 mdurl~=0.1.2
 mock~=5.0.2
 packaging~=23.1
-Pillow==10.3.0
+Pillow==10.4.0
```
Comment (michplunkett) [inline requirements.txt:46]: It looks like the Pillow fix for 3.13 has been implemented: https://github.com/pypa/setuptools/issues/4415

File: requirements.txt [hunk 7]
```diff
@@ -32,9 +32,9 @@ Flask-WTF==1.1.2
 google-api-python-client==2.92.0
 gunicorn==22.0.0
 httplib2==0.22.0
-idna~=3.4
+idna~=3.7
 itsdangerous~=2.1.2
-Jinja2~=3.1.2
+Jinja2~=3.1.3
```
Comment (michplunkett) [inline requirements.txt:37]: Screenshot 2024-10-23 at 10 10 05 AM

**Review by b-meson — APPROVED** [review]

**Pull request merged**